qa/rpc-tests/bip68-112-113-p2p.py

```diff
@@ -0,0 +1,547 @@
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+        for tx in bip68heighttxs:
+            yield TestInstance([[self.create_test_block([tx]), False]]) # 26 - 29
+
+        # Advance one block to 582
+        test_blocks = self.generate_blocks(1, 1234)
+        yield TestInstance(test_blocks, sync_every_block=False) # 30
+
+        # All BIP 68 txs should pass
+        bip68success_txs.extend(bip68heighttxs)
+        yield TestInstance([[self.create_test_block(bip68success_txs), True]]) # 31
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+
+
+        ### BIP 112 ###
+        ### Version 1 txs ###
+        # -1 OP_CSV tx should fail
+        yield TestInstance([[self.create_test_block([bip112tx_special_v1]), False]]) #32
+        # If SEQUENCE_LOCKTIME_DISABLE_FLAG is set in argument to OP_CSV, version 1 txs should still pass
+        success_txs = []
+        for b25 in xrange(2):
+            for b22 in xrange(2):
+                for b18 in xrange(2):
+                    success_txs.append(bip112txs_vary_OP_CSV_v1[1][b25][b22][b18])
+                    success_txs.append(bip112txs_vary_OP_CSV_9_v1[1][b25][b22][b18])
+        yield TestInstance([[self.create_test_block(success_txs), True]]) # 33
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+
+        # If SEQUENCE_LOCKTIME_DISABLE_FLAG is unset in argument to OP_CSV, version 1 txs should now fail
+        fail_txs = []
+        fail_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_v1))
+        fail_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_9_v1))
+        for b25 in xrange(2):
+            for b22 in xrange(2):
+                for b18 in xrange(2):
+                    fail_txs.append(bip112txs_vary_OP_CSV_v1[0][b25][b22][b18])
+                    fail_txs.append(bip112txs_vary_OP_CSV_9_v1[0][b25][b22][b18])
+
+        for tx in fail_txs:
+            yield TestInstance([[self.create_test_block([tx]), False]]) # 34 - 81
+
+        ### Version 2 txs ###
+        # -1 OP_CSV tx should fail
+        yield TestInstance([[self.create_test_block([bip112tx_special_v2]), False]]) #82
+
+        # If SEQUENCE_LOCKTIME_DISABLE_FLAG is set in argument to OP_CSV, version 2 txs should pass (all sequence locks are met)
+        success_txs = []
+        for b25 in xrange(2):
+            for b22 in xrange(2):
+                for b18 in xrange(2):
+                    success_txs.append(bip112txs_vary_OP_CSV_v2[1][b25][b22][b18]) # 8/16 of vary_OP_CSV
+                    success_txs.append(bip112txs_vary_OP_CSV_9_v2[1][b25][b22][b18]) # 8/16 of vary_OP_CSV_9
+
+        yield TestInstance([[self.create_test_block(success_txs), True]]) # 83
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+
+        ## SEQUENCE_LOCKTIME_DISABLE_FLAG is unset in argument to OP_CSV for all remaining txs ##
+        # All txs with nSequence 11 should fail either due to earlier mismatch or failing the CSV check
```

Comment:
nit: I think this comment should be `... nSequence 9 ...`, rather than `nSequence 11`?